src/Service.js

```diff
@@ -687,13 +723,14 @@ Service_.prototype.isExpired_ = function(token) {
  */
 Service_.prototype.canRefresh_ = function(token) {
   if (!token.refresh_token) return false;
-  var expiresIn = token.refresh_token_expires_in;
-  if (!expiresIn) {
-    return true;
+  this.setExpiresAt_(token);
+  if (!token.refreshTokenExpiresAt) {
+    return false;
```

Comment:
Shouldn't this be `return true` to match current logic? Tokens with no explicit refresh token expiry should be considered non-expiring and refreshable.

Reply:
Yep missed that. Fixed.